docs/themes/site/js/data-fill-from.js

```diff
@@ -5,7 +5,7 @@ $(document).ready(function() {
     var codeContainer = $this.data("fill-from");
     var $container = $(codeContainer);
 
-    var code = $.trim($container.html());
+    var code = $container.html().trim();
```

Comment:
https://api.jquery.com/jQuery.trim/

$.trim() handles nulls and undefined.  Should this also handle that?

Reply:
Yep, that seems worthwhile - it does seem that `.html()` can return `undefined` if the container element is not found in the DOM.